### PR TITLE
Add template to the widgets

### PR DESCRIPTION
**Pull request by bjorvack**
Make it possible to have different templates for different widget.

### CODE REVIEW

**Comment by StijnVrolijk** [comment]
Can you give a short explanation about what this does?

File: src/Backend/Modules/Partners/Domain/Widget/Widget.php [hunk 1]
```diff
@@ -146,6 +185,31 @@ function (PartnerDataTransferObject $partnerDataTransferObject) use ($widget) {
     }
 
     /**
+     * Get templates.
+     *
+     * @return array
+     */
+    public static function getTemplates()
```
Comment (carakas) [inline src/Backend/Modules/Partners/Domain/Widget/Widget.php:192]: I would move this to the repository, this has NO place in the entity

File: src/Backend/Modules/Partners/Domain/Widget/Widget.php [hunk 2]
```diff
@@ -115,6 +125,34 @@ public function getTitle()
...
+     */
+    public function getTemplate()
+    {
+        return $this->template;
+    }
+
+    /**
+     * @return string
+     */
+    public function getTemplatePath()
```
Comment (carakas) [inline src/Backend/Modules/Partners/Domain/Widget/Widget.php:138]: this belongs in the repository, not in the entity

**Review by carakas — CHANGES_REQUESTED** [review]

**Comment by carakas** [comment]
@StijnVrolijk it makes it possible to have multiple templates, just like in content blocks

**Comment by carakas** [comment]
@bjorvack see content blocks module in fork how to do it the proper way

**Pull request closed without merging**